cylc/flow/task_events_mgr.py

```diff
@@ -1327,7 +1334,9 @@ def _process_message_failed(self, itask, event_time, message, forced):
             timer = itask.try_timers[TimerFlags.EXECUTION_RETRY]
             self._retry_task(itask, timer.timeout)
             delay_msg = f"retrying in {timer.delay_timeout_as_str()}"
-            LOG.warning(f"[{itask}] {delay_msg}")
+            LOG.warning(
+                f'[{itask}] => {TASK_OUTPUT_FAILED} with {full_message}'
+                f' \n({delay_msg})')
```

Comment:
My preference for this log message would be
```suggestion
                f'[{itask}] {full_message or self.EVENT_FAILED} - '
                f'{delay_msg}'
            )
```